src/server/DBMetadata/columnTypeTranslators.js

```diff
@@ -0,0 +1,71 @@
+function pgSQL(type) {
```

Comment:
Does this column type translator support all pg and ms data types?

Reply:
@laueian The mssql supports all the common mssql value types, but I didn't check the pgsql one. I added checking the pgsql one as a separate ticket.